PurchasesCoreSwift/Caching/InMemoryCachedObject.swift

```diff
@@ -6,9 +6,7 @@
 //  Copyright © 2021 Purchases. All rights reserved.
 //
 
-import Foundation
-
-open class InMemoryCachedObject<T> {
+public class InMemoryCachedObject<T> {
 
     private let accessQueue = DispatchQueue(label: "InMemoryCachedObjectQueue", attributes: .concurrent)
     private var lastUpdated: Date?
```

Comment:
There was a comment here https://github.com/RevenueCat/purchases-ios/pull/658#discussion_r674338482 that I believe is unaddressed

Reply:
Oh yeah! I'm used to doing `private` vars/lets at the top for some reason, hard habit to break.

Reply:
Fixed!